public/app/plugins/datasource/prometheus/dataquery.cue

```diff
@@ -45,6 +45,10 @@ composableKinds: DataQuery: {
 				// See https://github.com/grafana/grafana/issues/48081
 				intervalFactor?: number
 
+				scope?: {
+					matchers: string
+				}
+
```

Comment:
Can we get rid of this cue file? I don't think having it makes sense anymore. WDYT?